.github/workflows/build-test-prValidation.yml

```diff
@@ -0,0 +1,64 @@
+name: PRValidation Workflow
+
+on:
+  pull_request:
+  push:
+    branches: [ master ]
+    tags-ignore: [ v* ]
+
+jobs:
+  check-code-style:
+    name: Checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: olafurpg/setup-scala@v10
+        with:
+          java-version: adopt@1.8.0
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.2
+
+#    ghtag "$ghprbPullId" akka akka '["validating"]' "tested needs-attention"
+
+      - name: Code style check
+        run: |-
+          # MiMa is disabled in GHActions because it's already covered in Travis builds (travis runs for multiple scala versions)
+          sbt -jvm-opts .jvmopts-ci \
+          -Dakka.ci-server=true \
+          -Dakka.test.tags.exclude=performance,timing,long-running \
+          -Dakka.test.multi-in-test=false \
+          -Dakka.test.timefactor=1 \
+          -Dakka.cluster.assert=on \
+          -Dsbt.override.build.repos=false \
+          -Dakka.test.multi-node=false \
+          -Dsbt.log.noformat=false \
+          -Dakka.log.timestamps=true \
+          scalafmtCheckAll scalafmtSbtCheck test:compile
+
+      - name: Akka validatePR
+        run: |-
+          # MiMa is disabled in GHActions because it's already covered in Travis builds (travis runs for multiple scala versions)
+          sbt -jvm-opts .jvmopts-ci \
+          -Dakka.mima.enabled=false \
+          -Dakka.ci-server=true \
+          -Dakka.test.tags.exclude=performance,timing,long-running \
+          -Dakka.test.multi-in-test=false \
+          -Dakka.test.timefactor=1 \
+          -Dakka.cluster.assert=on \
+          -Dsbt.override.build.repos=false \
+          -Dakka.test.multi-node=false \
+          -Dsbt.log.noformat=false \
+          -Dakka.log.timestamps=true \
+           publishLocal publishM2 validatePullRequest
```

Comment:
Is validatePullRequest what we want to do for pushing to master?

Reply:
😅 too much copy/paste.